integreat_compass/cms/templates/components/decline_list.html

```diff
@@ -0,0 +1,28 @@
+{% extends "_base.html" %}
+{% load i18n %}
+{% load static %}
+{% load widget_tweaks %}
+{% load render_bundle from webpack_loader %}
+{% block title %}
+    {% translate "Declined" %}
+{% endblock title %}
+{% block content %}
+    {% include "includes/dashboard_navbar.html" %}
+    {% for offer in declined_offers %}
+        <div class="flex flex-row border-2 border-gray-200 bg-white rounded-lg shadow-md shadow-inner p-6 mb-8">
+            <div class="basis-2/3 pr-8">
+                <p class="text-xl font-bold">{{ offer.title }}</p>
+                <p>{{ offer.offer.organization.name }}</p>
+                <p class="text-xl text-gray-700 font-semibold pt-4">{% translate "Short description" %}</p>
+                <p class="text-gray-600">{{ offer.description|truncatechars:500 }}</p>
+                <div class=" my-4">
+                    {% for tag in offer.offer.tags.all %}
+                        <span class="bg-red-500 text-white rounded-full py-1 px-2">{{ tag }}</span>
+                    {% endfor %}
+                </div>
+                <p class="text-gray-700">{% translate "Created at:" %} {{ offer.created_at|date:"d. M Y" }}</p>
+            </div>
+            <div class="basis-1/3 flex items-center"></div>
+        </div>
+    {% endfor %}
```

Comment:
```suggestion
    {% empty %}
        {% translate "No offers have been declined by the board." %}
    {% endfor %}
```

(or something along the lines)